shell/platform/android/io/flutter/embedding/engine/android/FlutterView.java

```diff
@@ -443,12 +444,15 @@ public void attachToFlutterEngine(@NonNull FlutterEngine flutterEngine) {
         textInputPlugin
     );
     androidTouchProcessor = new AndroidTouchProcessor(this.flutterEngine.getRenderer());
+    PlatformViewsController platformViewsController = flutterEngine.getPluginRegistry().getPlatformViewsController();
     accessibilityBridge = new AccessibilityBridge(
         this,
         flutterEngine.getAccessibilityChannel(),
         (AccessibilityManager) getContext().getSystemService(Context.ACCESSIBILITY_SERVICE),
-        getContext().getContentResolver()
+        getContext().getContentResolver(),
+        platformViewsController
     );
+    platformViewsController.attachAccessibilityBridge(accessibilityBridge);
```

Comment:
Do we need to detach this ever?

Reply:
I tried to keep the same lifespan as the AccessibilityBridge reference, should `accessibilityBridge` be unset in the new FlutterView like in the current one?

Reply:
@matthew-carroll May have more context to answer that question.